package.json

```diff
@@ -242,14 +253,15 @@
     "pretest": "npm run compile-tests && npm run lint",
     "test": "node ./out/test/runTest.js",
     "test:coverage": "c8 npm run test",
-    "fetch-api": "curl https://artifacts.codacy.com/api/codacy-api/33.2.15/apiv3.yaml -o ./api-v3/api-swagger.yaml",
+    "fetch-api": "curl https://artifacts.codacy.com/api/codacy-api/33.11.11/apiv3.yaml -o ./api-v3/api-swagger.yaml",
     "generate-api": "rm -rf ./src/api/client && openapi --input ./api-v3/api-swagger.yaml --output ./src/api/client --useUnionTypes --indent 2 --client axios",
     "update-api": "npm run fetch-api && npm run generate-api",
     "postinstall": "npm run update-api",
     "check-types": "tsc --noEmit"
   },
   "devDependencies": {
     "@codacy/openapi-typescript-codegen": "^0.0.2",
+    "@openapitools/openapi-generator-cli": "^2.7.0",
```

Comment:
was this one added for something? is it used? needed?

Reply:
removed - part of my fiddling around getting the updated api version